dace/frontend/python/replacements.py

```diff
@@ -2688,7 +2713,24 @@ def to_string(idx):
         output_indices.append(get_idx(i))
 
         not_none_dims = [d for d in dims if d is not None]
-        max_dim = max(not_none_dims)
+        # max_dim = max(not_none_dims)
+        max_dim = 0
+        for d in not_none_dims:
+            if isinstance(max_dim, Number):
+                if isinstance(d, Number):
+                    max_dim = max(max_dim, d)
+                elif symbolic.issymbolic(d):
+                    max_dim = d
+                else:
+                    raise NotImplementedError
+            elif symbolic.issymbolic(max_dim):
+                if isinstance(d, Number):
+                    pass
```

Comment:
This assumes that any symbol is larger than any number. We have a config for that.

The whole new section could use some documentation comment specifying the intended behavior of finding this max dim.

Reply:
Which is this config option?